python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -40,26 +41,56 @@
 )
 
 _LOCAL_HOST = "127.0.0.1"
-_TIMEOUT = 20  # timeout in s to be used by this module
+_TIMEOUT = 20  # timeout [sec] to be used by this module
 _KEYS_TO_REMOVE = {
     "status",
     "operationalMode",  # Remove when next XML release is done
     "appliedConfiguration",  # Remove when next XML release is done
 }
-_KEYS_IN_RADIANS = {
+
+# The values of these keys need to be compensated for the dome azimuth offset
+# in the AMCS status. Note that these keys are shared with LWSCS so they can be
+# added to _KEYS_IN_RADIANS to avoid duplication but this also means that an
+# additional check for the AMCS lower level component needs to be done when
+# applying the offset correction. That is a trade off I can live with.
+_AMCS_KEYS_OFFSET = {
     "positionActual",
     "positionCommanded",
+}
+# The values of these keys need to be converted from radians to degrees when
+# the status is recevied and telemetry with these values is sent.
+_KEYS_IN_RADIANS = {
     "velocityActual",
     "velocityCommanded",
-}
+}.union(_AMCS_KEYS_OFFSET)
+
+# The offset of the dome rotation zero point with respect to azimuth 0º (true
+# north) is 32º west and the aperture lies at the opposite side of the dome.
+# Therefore the dome azimuth offset is 180º - 32º = 148º and this needs to be
+# subtracted when commanding the azimuth position, or added when sending the
+# azimuth telemetry.
+DOME_AZIMUTH_OFFSET = 148.0
 
+# Polling periods [sec] for the lower level components.
 _AMCS_STATUS_PERIOD = 0.2
 _APsCS_STATUS_PERIOD = 2.0
 _LCS_STATUS_PERIOD = 2.0
 _LWSCS_STATUS_PERIOD = 2.0
 _MONCS_STATUS_PERIOD = 2.0
 _THCS_STATUS_PERIOD = 2.0
 
+# These next commands are temporarily disabled because they will be issued
+# during the upcoming TMA pointing test and the EIE LabVIEW code doesn't handle
+# them yet, which will result in an error. As soon as the TMA pointing test is
+# done, they will be reenabled.
+COMMANDS_DISABLED_FOR_TMA_POINTING = {
+    "crawlEl",
+    "goStationaryEl",
+    "moveEl",
+    "stopEl",
+}
+DATA_FOR_TMA_POINING = {"response": 0, "timeout": 0}
```

Comment:
I gently suggest a name like REPLY_DATA_FOR_DISABLED_COMMANDS. It took me awhile to figure out what it was.

If you keep the name, I suspect POINING should be POINTING.

Reply:
Good suggestion. I renamed it.